ElectronicShop/Models/LoadShop.cs

```diff
@@ -0,0 +1,27 @@
+﻿
+
+using ElectronicShop.Models.Interfaces;
+
+namespace ElectronicShop.Models
+{
+    internal class LoadShop : ILoadShop
+    {
+        //1. SingUp User 2. LogIn User 3. Login Admin 4. Exit
+
+        void ILoadShop.LoadShop(out int loadSelect)// sioje vietoje (out int loadSelect) kur daryti SWITCH
```

Comment:
Nesu tikra, bet man atrodo, kad nereikia pilnai rašyti `void ILoadShop.LoadShop` užtenka be interface pavadinimo `void LoadShop`